### PR TITLE
PUBDEV-9062: Stacked Ensembles algorithm parameter reorganization

### PR DESCRIPTION
**Pull request by hannah-tillman**
For: [PUBDEV-9062](https://h2oai.atlassian.net/browse/PUBDEV-9062) (Sub-task of [PUBDEV-8049](https://h2oai.atlassian.net/browse/PUBDEV-8049))

This PR reorganizes Stacked Ensembles parameters by algorithm-specific and common.

It also updates the parameter descriptions to help standardize the structure amongst each parameter.

[PUBDEV-9062]: https://h2oai.atlassian.net/browse/PUBDEV-9062?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PUBDEV-8049]: https://h2oai.atlassian.net/browse/PUBDEV-8049?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: h2o-docs/src/product/data-science/stacked-ensembles.rst [hunk 1]
```diff
@@ -70,55 +70,79 @@ Before training a stacked ensemble, you will need to train and cross-validate a
...
 
--  `validation_frame <algo-params/validation_frame.html>`__: (Optional) Specify the dataset to use for tuning the model.  The validation frame will be passed through to the metalearner for tuning.
+-  `blending_frame <algo-params/blending_frame.html>`__: Specify a frame to be used for computing the predictions that serve as the training frame for the metalearner. This triggers blending mode if provided.
 
--  `blending_frame <algo-params/blending_frame.html>`__: (Optional) Specify a frame to be used for computing the predictions that serve as the training frame for the metalearner. This triggers blending mode if provided.
+-  `metalearner_algorithm <algo-params/metalearner_algorithm.html>`__ Specify the metalearner algorithm type. Options include:
```
Comment (tomasfryda) [inline h2o-docs/src/product/data-science/stacked-ensembles.rst:82]:
```suggestion
-  `metalearner_algorithm <algo-params/metalearner_algorithm.html>`__: Specify the metalearner algorithm type. Options include:
```
missing colon

File: h2o-docs/src/product/data-science/stacked-ensembles.rst [hunk 2]
```diff
@@ -70,55 +70,79 @@ Before training a stacked ensemble, you will need to train and cross-validate a
...
--  `metalearner_transform <algo-params/metalearner_transform.html>`__: (Optional) Specify the transformation used on predictions from the base models in order to make a level one frame. Options include:
+-  **keep_levelone_frame**: Keep the level one data frame that's constructed for the metalearning step. This option defaults to ``False`` (disabled).
 
- - ``"NONE"`` (no transform applied)
- - ``"Logit"`` (applicable only to classification tasks, use logit transformation on the predicted probabilities)
+Common parameters
+'''''''''''''''''
 
--  `offset_column <algo-params/offset_column.html>`__: (Optional; Availability depends on the ``metalearner_algorithm``) Specify a column to use as the offset.
+-  `training_frame <algo-params/training_frame.html>`__ *Required* Specify the dataset used to build the model. In a Stacked Ensemble model, the training frame is used only to retreive the response column (needed for training the metalearner) and also to compute training metrics for the ensemble model.
```
Comment (tomasfryda) [inline h2o-docs/src/product/data-science/stacked-ensembles.rst:117]:
```suggestion
-  `training_frame <algo-params/training_frame.html>`__: *Required* Specify the dataset used to build the model. In a Stacked Ensemble model, the training frame is used only to retreive the response column (needed for training the metalearner) and also to compute training metrics for the ensemble model.  
```
missing colon

Comment (hannah-tillman) [reply h2o-docs/src/product/data-science/stacked-ensembles.rst]: Thank you for finding those! :)

**Review by ledell — APPROVED** [review]
Thank you, @hannah-tillman!

**Review by tomasfryda — APPROVED** [review]

**Pull request merged**